app-template/sitemap/generate-sitemap.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs')
+fs.readFile('src/config/routesConfig.ts', (err, data) => {
+  if (err) {
+    return console.error(err)
+  }
+  let routesConfigContent = data.toString()
+  //change baseUrl depending on the portal domain
+  const args = process.argv.slice(2)
```

Comment:
I think it would help if there was a comment stating that the file expects a domain name as input

Reply:
added missing usage error that talks about the required argument